### PR TITLE
pythonPackages.push-receiver: init at 0.1.1

### PR DESCRIPTION
**Pull request by veehaitch**
###### Motivation for this change

https://github.com/Francesco149/push_receiver

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by veehaitch** [comment]
Result of `nixpkgs-review pr 130078` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.push_receiver</li>
    <li>python39Packages.push_receiver</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 130078` at 8fa9b163 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.push_receiver</li>
    <li>python39Packages.push_receiver</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/development/python-modules/push_receiver/default.nix:31:0:
```
   |
31 |     description = "Subscribe to GCM/FCM and receive notifications";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -5645,6 +5645,8 @@ in {
 
   purl = callPackage ../development/python-modules/purl { };
 
+  push_receiver = callPackage ../development/python-modules/push_receiver { };
```
Comment (jonringer) [inline pkgs/top-level/python-packages.nix:5648]: Please normalize the attribute name. According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)
```suggestion
  push-receiver = callPackage ../development/python-modules/push-receiver { };
```

Comment (veehaitch) [reply pkgs/top-level/python-packages.nix]: Thanks, done!

**Pull request merged**